spec/anthropic_spec.rb

```diff
@@ -1,9 +1,10 @@
 # frozen_string_literal: true
 
 require "webmock/rspec"
+require "llm/providers/anthropic"
 
 RSpec.describe LLM::Anthropic do
```

Comment:
@0x1eef Do you think it's correct to require the provider in the spec file?

Reply:
This could be:
```ruby
RSpec.describe "LLM::Anthropic" do
  ..
end
```
Then you shouldn't need the require anymore.